dtls_debug.c

```diff
@@ -252,7 +252,7 @@ dsrv_log(log_t level, const char *format, ...) {
   va_end(ap);
   fflush(log_fd);
 }
-#elif defined (HAVE_VPRINTF) /* WITH_CONTIKI */
+#elif defined (HAVE_VPRINTF) || defined(IS_WINDOWS) /* WITH_CONTIKI */
```

Comment:
Does cmake not detect that on windows automatically?

At least, "AutoConf.cmake" contains `check_function_exists (vprintf        HAVE_VPRINTF)` which should to the job.